elyra/cli/pipeline_app.py

```diff
@@ -155,6 +160,8 @@ def _preprocess_pipeline(pipeline_path: str,
         primary_pipeline["app_data"]["runtime"] = runtime
     if runtime_config:
         primary_pipeline["app_data"]["runtime-config"] = runtime_config
+    else:
```

Comment:
I still don't feel comfortable defaulting here... have you seen a case where we are not passing the proper 'local' for local runs?